example/src/App.tsx

```diff
@@ -78,7 +77,7 @@ const App = () => {
       />
       {enablePolymerEditor && <PolymerToggler toggle={setShowPolymerEditor} />}
       {hasError && (
-        <ErrorModal
+        <InfoModal
```

Comment:
Good job on moving `ErrorModal` into `ketcher-react/` and good naming. However, I'd suggest using `<InfoModal />` somewhere in `ketcher-react/` instead of `example/`. Other parts are great, and the business requirement is fulfilled, good job!

Reply:
I totally agree. I think this is the responsibility of Ketcher to show some warning, not the client app. Although, Ketcher can emit some event that error has occurred. And client code can react (or not) on it somehow.

Reply:
Hi @yuleicul @Nitvex thank you for comments! Should the error / info modal be triggered through ErrorsContext.Provider somehow? Currently Ketcher has errorHandler in Class Editor and error handling mechanism implemented [in PR](https://github.com/epam/ketcher/pull/730/files) , should it be extended by displaying modal from Ketcher?  There is also a Dialog component, should it be used as well for consistency? (as in other modals in the app)

Reply:
Hi @Inga-Vishnivetskaia , I found it was not as easy as I expected before.

I found two usages of `errorHandle` in our codebase:
- Get `errorHandle` from `ErrorsContext.Provider`, and use it
- Get `errorHanlde` from `state.editor.errorHanlde`, and use it

My current solution (maybe not the best) is to get `errorHandle` from `ErrorsContext.Provider` everywhere, then it would be easier to move `InfoModal` to `ketcher-react/`.

For easy review, I'd suggest creating another issue to refactor this part. Maybe the name can be 'Refactoring request: use `InfoModal` from `example/` to `ketcher-react/`'.

Reply:
Hi @Nitvex currently Paste / Cut / Copy modals are triggered only by library. 
1) Should all other error handling (with modal display) also be controlled / moved to the library (still leaving errorHandler method available for client)?
2) Should text for Cut / Copy modals also be made specific (with shortcut display)?